containerd-shim-v2/wait.go

```diff
@@ -19,6 +19,8 @@ import (
 	"google.golang.org/grpc/codes"
 )
 
+const defaultCheckInterval = 1 * time.Second
```

Comment:
@liubin Is this `defaultCheckInterval` by any chance related to the `defaultCheckInterval` in package virtcontainers (in `monitor.go`)?  If so, wouldn't it be more maintainable to reference the original value instead of redefining it here?

Reply:
Hi @pmores , you  are right, I have updated it.

Reply:
Thanks @liubin for the swift reaction!

Reply:
Let me mark this one as resolved.